packages/botonic-react/src/webchat/cover-component/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from 'react'
+
+import { WEBCHAT } from '../../constants'
+import { WebchatContext } from '../../contexts'
+
+interface Props {
```

Comment:
rename to `CoverComponentProps`

Reply:
CoverComponentProps already exists elsewhere in the code

Reply:
so we'll need to refactor this in the future, we should not have it twice. Let's do it in next iterations as we simplify the code